src/cmps/tag/Tag.css

```diff
@@ -1,23 +1,21 @@
-.tag-container{
+.tag-container {
     background-color: rgb(189, 189, 189);
     margin: 2px;
     padding: 5px;
     display: flex;
     justify-content: space-between;
-    border-radius: 50px;    
+    border-radius: 50px;
```

Comment:
do you mean 50%?

Reply:
% = Creates a radius from the center of the object
px = Creates a radius only at the edges of the object